### DISCUSSION TITLE
Is it possible to turn off the progress bar?

### DISCUSSION THREAD

**Discussion by Komal-99** [Data-Centric-AI-Community/fg-data-profiling#997] [Q&A] [answered]
Hi, 
Is there any way by which we can stop  printing these downloading  lines  In the python terminal,  which comes whenever we use the to_file() function to save the ProfileReport because I am integrating my ML model with one of the Software and It can't run If there is any output in terminal other that Answer of my model so I want to hide them or remove them from displaying these in terminal and directly saving my ProfilePreport into a HTML file

Please help me out it's urgent need
![image](https://user-images.githubusercontent.com/74819092/173898726-addcb495-4adb-4f19-8fcc-0bed9c3ee490.png)

**Comment by sbrugman** [comment, marked as answer]
Not sure if I understand your question correctly, but it's possible to disable the progress bar by passing "progress_bar=False".

**Comment by Komal-99** [comment]
Hi ,
Yes  you got it right , Thanks for the solution
Regards
Komal
